cor-journey/src/main/scala/essttp/rootmodel/AmountInPence.scala

```
@@ -40,10 +40,10 @@ object AmountInPence {
   def apply(str: String): AmountInPence = {
     str match {
       case s if s.isEmpty => AmountInPence(0)
-      case s              => AmountInPence(BigDecimal(s).longValue() * 100)
+      case s              => AmountInPence((BigDecimal(s) * 100).longValue())
     }
   }
-  def apply(bigDecimal: BigDecimal): AmountInPence = AmountInPence(bigDecimal.longValue() * 100)
+  def apply(bigDecimal: BigDecimal): AmountInPence = AmountInPence((bigDecimal * 100).longValue())
```

failing silently with `longValue` seems very dangerous - it'd be safer to use `tolongExact` and have it throw if the number can't actually fit in a `Long`

---

will update